### ISSUE TITLE
[1.18.2] Radium throws IllegalAccessError when loading "any" mod fails

### ISSUE THREAD

**Issue by tobi1449** [Asek3/Radium#57]
When any mod fails loading (e.g. because its dependencies are unmet), the following error is thrown instead of the real one: 

```
Caused by: java.lang.IllegalAccessError: class me.jellysquid.mods.lithium.common.shapes.VoxelShapeSimpleCube tried to access method 'void net.minecraft.world.phys.shapes.VoxelShape.<init>(net.minecraft.world.phys.shapes.DiscreteVoxelShape)' (me.jellysquid.mods.lithium.common.shapes.VoxelShapeSimpleCube is in module radium@0.7.10 of loader 'TRANSFORMER' @6af78a48; net.minecraft.world.phys.shapes.VoxelShape is in module minecraft@1.18.2 of loader 'TRANSFORMER' @6af78a48)
	at TRANSFORMER/radium@0.7.10/me.jellysquid.mods.lithium.common.shapes.VoxelShapeSimpleCube.<init>(VoxelShapeSimpleCube.java:31)
```

**Comment by Corvus20** [comment]
Wait this is what that means?
I'm gonna have to go through my modlist and see if I have anything missing.

**Comment by tobi1449** [comment]
Just disable Radium, start Minecraft, fix whatever's broken and enable Radium again.

**Comment by falxie** [comment]
I ran into this when accidentally installing a 1.19 mod in 1.18. Very confusing since it makes you think there's an issue between the mod and Radium not the mod and something else.

**Comment by falxie** [comment]
This was reported for the 1.18.2 but this only seems fixed for 1.19.2. Is the 1.18.2 port unmaintained?
